Classes/Configuration/ExtensionConfiguration.php

```diff
@@ -45,7 +42,7 @@ private static function init()
     {
         if (!self::$isInitialized) {
             self::$isInitialized = true;
-            $extensionConfig = GeneralUtility::makeInstance(ExtensionConfigurationCore::class)->get('fal_securedownload');
+            $extensionConfig = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Configuration\ExtensionConfiguration::class)->get('fal_securedownload');
```

Comment:
Just out of curiosity: why do we use the full namespace here and remove the use statement above?